src/main/kotlin/no/nav/familie/ba/sak/kjerne/beregning/TilkjentYtelseUtils.kt

```diff
@@ -154,24 +163,31 @@ object TilkjentYtelseUtils {
                                         }
 
                                 val beløpsperioder =
-                                        listOf(beløpsperioderFørFylte6År, beløpsperioderEtterFylte6År).flatten()
+                                        listOf(satsperioderFørFylte6År, beløpsperioderEtterFylte6År).flatten()
                                                 .sortedBy { it.fraOgMed }
                                                 .fold(mutableListOf(), ::slåSammenEtterfølgendePerioderMedSammeBeløp)
 
                                 beløpsperioder.map { beløpsperiode ->
+                                    val prosent = finnProsentForAndel(fom = beløpsperiode.fraOgMed,
+                                                                      tom = beløpsperiode.tilOgMed,
+                                                                      endringerForPerson = endredeUtbetalingsandeler?.filter { it.person.personIdent.ident == person.personIdent.ident })
                                     AndelTilkjentYtelse(
                                             behandlingId = vilkårsvurdering.behandling.id,
                                             tilkjentYtelse = tilkjentYtelse,
                                             personIdent = person.personIdent.ident,
                                             stønadFom = beløpsperiode.fraOgMed,
                                             stønadTom = beløpsperiode.tilOgMed,
-                                            beløp = beløpsperiode.beløp,
-                                            type = YtelseType.ORDINÆR_BARNETRYGD
+                                            kalkulertUtbetalingsbeløp = beløpsperiode.sats.toBigDecimal()
+                                                    .times(prosent)
+                                                    .divide(100.toBigDecimal())
+                                                    .toInt(),
+                                            type = finnYtelseType(behandling.kategori, behandling.underkategori, person.type),
+                                            sats = beløpsperiode.sats,
+                                            prosent = prosent,
```

Comment:
```suggestion
                                            prosent = beløpsperiode.prosent,
```

https://github.com/navikt/familie-ba-sak/pull/1415/files#r712943558
https://github.com/navikt/familie-ba-sak/pull/1415/files#r712996744

Som @MarcusGustafson74 og @henninghaakonsen  er inne på vil 50% for delt bosted-vilkår settes tidligere, og lenger opp her hvor man tidligere har satt 1024kr eller 527kr basert på en utledning vil vi nå sette prosent etc. 
Håndtering av overstyringer gjør vi separat i det vi implementerte sist, så jeg er faktisk ikke sikker på om man trenger å ta inn `endredeUtbetalingsandeler: List<EndretUtbetalingAndel>` i `beregnTilkjentYtelse` i det hele tatt? :)